SOURCES/secureboot-certs

```diff
@@ -708,7 +708,7 @@ be passed to {} as custom auth files.
 """.format(
             os.path.basename(sys.argv[0])
         ),
-        default='default',
+        default='none',
```

Comment:
Could you undo this change? Updating getdefault() should be enough, this should remain default so that we don't forget to change it later if we bring back dbx.auth. Thanks